packages/webdriverio/src/utils/findStrategy.js

```diff
@@ -62,6 +62,10 @@ const defineStrategy = function (selector) {
     if (selector.startsWith('~')) {
         return 'accessibility id'
     }
+    // Recursive element search using image content
+    if (selector.startsWith('appium=')) {
```

Comment:
I find `appium` a bit too generic, how about:

- `$('/path/to/image.png')`: if selector ends with an image suffix
- `$('🖼=/path/to/image.png')`: maybe a bit too much
- `$('img=/path/to/image.png')`

I personally prefer the first one.

Reply:
sure, will give it a try to use the first one